specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/virtualWan.json

```diff
@@ -5034,6 +5034,237 @@
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the NVA connection."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. Returns the details of the NVA connection",
+            "schema": {
+              "$ref": "#/definitions/NetworkVirtualApplianceConnection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "NetworkVirtualApplianceConnections_Delete",
+        "x-ms-examples": {
+          "NetworkVirtualApplianceConnectionDelete": {
+            "$ref": "./examples/NetworkVirtualApplianceConnectionDelete.json"
+          }
+        },
+        "description": "Deletes a NVA connection.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "networkVirtualApplianceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Network Virtual Appliance."
+          },
+          {
+            "name": "connectionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the NVA connection."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Delete successful.NVA Connection deleted."
+          },
+          "202": {
+            "description": "Accepted, and the operation will continue asynchronously."
+          },
+          "204": {
+            "description": "No NVA connections exist by the name provided."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkVirtualAppliances/{networkVirtualApplianceName}/networkVirtualApplianceConnections": {
```

Comment:
>networkVirtualApplianceConnections [](http://example.com/codeflow?start=155&length=34)

Make 'NetworkVirtualApplianceConnections_List' Operation pageable. Consider adding the x-ms-pageable extension.

Reply:
I have updated.